www/modules/centreon-open-tickets/class/rule.php

```diff
@@ -349,35 +420,48 @@ public function duplicate($select = array(), $duplicateNb = array()) {
             if (isset($duplicateNb[$ruleId]) && $duplicateNb[$ruleId] > 0) {
                 for ($j = 1; $j <= $duplicateNb[$ruleId]; $j++) {
                     $name = $row['alias'] . "_" . $j;
-                    $res2 = $this->_db->query("SELECT `rule_id` FROM `mod_open_tickets_rule` WHERE `alias` = '" . $this->_db->escape($name) . "'");
+                    $res2 = $this->_db->query(
+                        "SELECT `rule_id` FROM `mod_open_tickets_rule` WHERE `alias` = '" .
+                        $this->_db->escape($name) . "'"
+                    );
                     while ($res2->rowCount()) {
                         $res2->free();
                         $i++;
                         $name = $row['alias'] . "_" . $i;
-                        $res2 = $this->_db->query("SELECT `rule_id` FROM `mod_open_tickets_rule` WHERE `alias` = '" . $this->_db->escape($name) . "'");
+                        $res2 = $this->_db->query(
+                            "SELECT `rule_id` FROM `mod_open_tickets_rule` WHERE `alias` = '" .
+                            $this->_db->escape($name) . "'"
+                        );
                     }
                     $query = "INSERT INTO mod_open_tickets_rule
-							(`alias`, `provider_id`, `activate`) VALUES " .
-							"('" . $this->_db->escape($name) . "', " . $row['provider_id'] . ", " . $row['activate'] . ")";
+                        (`alias`, `provider_id`, `activate`) VALUES " .
+                        "('" . $this->_db->escape($name) . "', " .
+                        $row['provider_id'] . ", " . $row['activate'] . "
+                    )";
                     $this->_db->query($query);
 
                     $nrule_id = $this->_db->lastinsertId('mod_open_tickets_rule');
 
                     // Duplicate form clone
-                    $res2 = $this->_db->query("SELECT * FROM mod_open_tickets_form_clone WHERE rule_id=$ruleId");
+                    $res2 = $this->_db->query("SELECT * FROM mod_open_tickets_form_clone WHERE rule_id=" . $ruleId);
                     while (($row2 = $res2->fetch())) {
                         $query = "INSERT INTO mod_open_tickets_form_clone
-								(`uniq_id`, `label`, `value`, `rule_id`, `order`) VALUES " .
-								"('" . $this->_db->escape($row2['uniq_id']) . "', '" . $this->_db->escape($row2['label']) . "', '" . $this->_db->escape($row2['value']) . "', " . $nrule_id . ", '" . $row2['order'] . "')";
+                            (`uniq_id`, `label`, `value`, `rule_id`, `order`) VALUES " .
```

Comment:
if don't you use concatenation, on line 448, why use it between lines 449 and 450